new/chef-workstation.md

```diff
@@ -0,0 +1,108 @@
+---
+RFC: unassigned
+Title: The Future of Chef Workstation
+Author:
+- Jon Morrow <jmorrow@chef.io>
+- Noah Kantrowitz <noah@coderanger.net>
+Status: Draft
+Type: Informational
+---
+
+# The Future of Chef Workstation
+
+ChefConf 2018 saw the unveiling of both the `chef-run` command and a Chef Workstation
+installer package containing it. This has lead to user confusion on two fronts.
+First, the term "Chef run" (or "Chef client run") is already in very common use
+in our community, so trying to communicate about `chef-run` has been tricky.
+Second, having two installers (Chef Workstation and ChefDK) has resulted in a
```

Comment:
👍 to not having 2 installers for the "workstation" experience (whatever that installer ends up called). I think thats one of my biggest motivations for this RFR - I want to converge on 1 product ASAP.